### PR TITLE
chore: add volunteer kaverigotkhindi

### PR DESCRIPTION
**Pull request by kaverigotkhindi**
Add new volunteer Kaveri Gotkhindi

### CODE REVIEW

File: data/users/kaverigotkhindi.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Kaveri",
+  "bio": "Web Developer",
+  "image": "https://pbs.twimg.com/profile_images/1488460804227997698/rPSir7Zj_400x400.jpg",
+  "categories": [
+    {
+      "slug": " web-development | software testing | SQL ",
+      "topics": ["HTML","CSS","Testing"," Databases","SQL"]
```
Comment (codewithvoid) [inline data/users/kaverigotkhindi.json:8]: Split tags in multiple topics. Max 2-3 per topic.

Comment (codewithvoid) [inline data/users/kaverigotkhindi.json:8]: this requires a new topic

**Comment by codewithvoid** [comment]
Thanks for volunteering Kaveri :)

**Comment by RakeshSangem** [comment]
Hey @kaverigotkhindi , Could you make changes as void mentioned above.

Please let us know if you need any further help:)

File: data/users/kaverigotkhindi.json (hunk 1 above)
Comment (kaverigotkhindi) [reply data/users/kaverigotkhindi.json:8]: I should make 2 seperate slugs ? one for web dev and another for SQL ? for each slug I should have topic ?

File: data/users/kaverigotkhindi.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Kaveri",
+  "bio": "Web Developer",
+  "image": "https://pbs.twimg.com/profile_images/1488460804227997698/rPSir7Zj_400x400.jpg",
+  "categories": [
+    {
+      "slug": " web-development",
+      "topics": ["HTML","CSS"]
+    },{
+      "slug": "SQL",
```
Comment (codewithvoid) [inline data/users/kaverigotkhindi.json:10]: Category slug should be from from [this file](https://github.com/codewithvoid/AskDevs/blob/main/public/categories.json)

```json
{
      "slug": "SQL",
      "topics": ["MSSQL", "Oracle SQL"]
}
```

File: data/users/kaverigotkhindi.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Kaveri",
+  "bio": "Web Developer",
+  "image": "https://pbs.twimg.com/profile_images/1488460804227997698/rPSir7Zj_400x400.jpg",
+  "categories": [
+    {
+      "slug": " web-development",
```
Comment (codewithvoid) [inline data/users/kaverigotkhindi.json:7]: remove extra space on line 7

File: data/users/kaverigotkhindi.json (hunk 2 above)
Comment (codewithvoid) [reply data/users/kaverigotkhindi.json:10]: Updated slug. My bad.

**Comment by RakeshSangem** [comment]
I think it's fine now. @codewithvoid TAL new changes.

**Comment by codewithvoid** [comment]
> 

Yes entries are fine, just json format needs fixing

**Comment by RakeshSangem** [comment]
> > 
> 
> Yes entries are fine, just json format needs fixing

I'll guide

**Review by RakeshSangem — APPROVED** [review]
LGTM🎉

Thanks for volunteering and for your patience!

File: data/users/kaverigotkhindi.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  "image":"https://pbs.twimg.com/profile_images/1488460804227997698/rPSir7Zj_400x400.jpg",
+  "categories":[
+    {
+      "slug":"web-development",
+      "topics":["HTML","CSS"]
+    },
+    {
+      "slug":"database-development",
+      "topics":["Microsoft SQL","Oracle SQL"]
+    },
```
Comment (codewithvoid) [inline data/users/kaverigotkhindi.json:13]: closing of square bracket is missing on this line. here's fixed json
```json
{
  "name": "Kaveri",
  "bio": "Web Developer",
  "image": "https://pbs.twimg.com/profile_images/1488460804227997698/rPSir7Zj_400x400.jpg",
  "categories": [
    {
      "slug": "web-development",
      "topics": [
        "HTML",
        "CSS"
      ]
    },
    {
      "slug": "database-development",
      "topics": [
        "Microsoft SQL",
        "Oracle SQL"
      ]
    }
  ],
  "contacts": {
    "twitter": "kaveri04"
  }
}
```

**Comment by codewithvoid** [comment]
Thanks for quick revert on review comments. Welcome!

**Comment by kaverigotkhindi** [comment]
Yey! Thank you Void and Rakesh for helping me out here . Appreciate your patience .

**Comment by RakeshSangem** [comment]
> Yey! Thank you Void and Rakesh for helping me out here . Appreciate your patience .

You are most welcome🙏

**Pull request merged**